package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "igniteui-angular-i18n",
+  "version": "1.0.0",
+  "description": "IgniteUI for Angular localization resources repository",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "lint": "tslint 'src/**/*.ts'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/IgniteUI/igniteui-angular-i18n.git"
+  },
+  "keywords": [
+    "IgniteUI",
+    "Angular",
+    "i18n",
+    "Infragistics"
+  ],
+  "author": "Infragistics",
+  "license": "Apache-2.0",
```

Comment:
Not sure what the licence should be. Could you please verify it's the same as `igniteui-angular`.